.changes/0.0.0.md

```diff
@@ -1,5 +1,6 @@
 ## Previous Releases
 For information on prior major and minor releases, see their changelogs:
+- [1.5](https://github.com/dbt-labs/dbt-spark/blob/1.5.latest/CHANGELOG.md)
```

Comment:
We should add this to `./CHANGELOG.md` as well.

Reply:
odd for some reason changie actually push up should be fixed now